tests/lean/ppExpr.lean.expected.out

```diff
@@ -1,3 +1,3 @@
 #0
-fun (a : Prop) => a
-id (@id Nat (id Nat.zero))
+fun a => a
+id (α := Nat) ((@id Nat (id (α := Nat) Nat.zero : Nat) : Nat) : Nat)
```

Comment:
Is this one expected? :smile:

Reply:
Fixed in https://github.com/dselsam/lean4/commit/d77208c5d7f3325cfd8bc213edea601b7dffc419 It turns out it was a bug in the test itself (`id` was given the wrong universe level).